src/ranking.jl

```diff
@@ -171,3 +171,173 @@ Missing values are assigned rank `missing`.
 """
 tiedrank(x::AbstractArray; sortkwargs...) =
     _rank(_tiedrank!, x, Float64; sortkwargs...)
+
+"""
+    quantilerank(v, value; method=:inc)
```

Comment:
Thinking about it, `v` is a weird name for a collection, and it's confusing since we also have `value`. Can you call it `itr` (in the docs and in the code), consistent with e.g. `sum` and `mean`?

Reply:
yes. I will change it

Reply:
Done in commit 3e30dd1.